euclidlike/instrument_params.py

```diff
@@ -124,6 +124,9 @@
 vis_blue_limit = 540
 vis_red_limit = 910
 
+# Coadd info
+coadd_zeropoint = 23.9  # Euclid zero-point in AB mag, used in all bands
```

Comment:
I realize it may seem silly but it would also be good to document the source for this